src/routes/(app)/asset/[filename]/+server.ts

```diff
@@ -1,22 +1,38 @@
 import { rootDir } from '$lib/server/root-dir';
+import { createHash } from 'crypto';
 import fs from 'fs';
 import { join } from 'path';
 
-export async function GET({ params }) {
+export async function GET({ params, request }) {
 	const assetDir = join(rootDir, 'src/lib/assets');
 	const filePath = join(assetDir, params.filename);
```

Comment:
I just realized we introduced a [path-traversal](https://owasp.org/www-community/attacks/Path_Traversal) vulnerability here.

I'll make a separate PR addressing this.